Static/DataSource.swift

```diff
@@ -204,7 +204,13 @@ extension DataSource: UITableViewDataSource {
     }
 
     public func tableView(_ tableView: UITableView, heightForHeaderInSection sectionIndex: Int) -> CGFloat {
-        return section(at: sectionIndex)?.header?.viewHeight ?? 0
+        guard let headerView = section(at: sectionIndex)?.header else { return 0 }
+
+        var headerHeight = headerView.viewHeight
+        if headerHeight == 0 {
```

Comment:
After reading through the documentation, I'm not so sure we should be setting this to automatic given value of zero. Why are we setting to auto here?

    open var sectionFooterHeight: CGFloat // default is UITableViewAutomaticDimension